packages/web-frontend/src/containers/MapOverlayBar/Control.js

```diff
@@ -114,28 +177,43 @@ export const Control = ({
         </Wrapper>
         <MapTableModal />
       </Header>
-      {isMapOverlaySelected ? (
-        selectedMapOverlays.map(mapOverlay => (
-          <TitleAndDatePicker
-            key={mapOverlay.mapOverlayCode}
-            mapOverlay={mapOverlay}
-            onUpdateOverlayPeriod={onUpdateOverlayPeriod}
-            isExpanded={isExpanded}
-            isMapOverlaySelected={isMapOverlaySelected}
-            toggleMeasures={toggleMeasures}
-            isMeasureLoading={isMeasureLoading}
-          />
-        ))
-      ) : (
-        <Content>
-          <EmptyContentText>{emptyMessage}</EmptyContentText>
-        </Content>
+      <DatePickerWrapper>
+        {isMapOverlaySelected ? (
+          reorderedSelectedMapOverlays.map((mapOverlay, index) => (
+            <div key={mapOverlay.mapOverlayCode}>
+              <TitleAndDatePicker
+                mapOverlay={mapOverlay}
+                onUpdateOverlayPeriod={onUpdateOverlayPeriod}
+                isMapOverlaySelected={isMapOverlaySelected}
+                isMeasureLoading={isMeasureLoading}
+                pinnedOverlay={pinnedOverlay}
+                setPinnedOverlay={setPinnedOverlay}
+              />
+              {index !== selectedMapOverlays.length - 1 && <Divider />}
```

Comment:
```suggestion
              {index < selectedMapOverlays.length - 1 && <Divider />}
```
It's more conventional to use less than to check for the not last items

Reply:
Love it. < is more readable.